apps/hash-api/src/graphql/create-apollo-server.ts

```diff
@@ -176,7 +176,10 @@ export const createApolloServer = ({
         },
       },
       ApolloServerPluginLandingPageGraphQLPlayground({
-        settings: { "request.credentials": "include" },
+        settings: {
+          "request.credentials": "include",
+          "schema.polling.enable": false,
```

Comment:
We don't need to poll the schema in the playground and it makes it extremely difficult to see anything going on in the Graph API at higher log levels